pre_commit_hooks/shellcheck_run_steps.py

```diff
@@ -109,6 +109,15 @@ def main(argv: Sequence[str] | None = None) -> int:
             "w",
             delete_on_close=False,
         ) as compiled_out:
+            try:
+                with open(filename) as pkg:
+                    yaml_def = yaml.load(pkg)
+                    architectures = yaml_def["package"].get("target-architecture", [])
+                    if not architectures:
+                        architectures = ["x86_64"]
+            except (FileNotFoundError, ruamel.yaml.YAMLError):
+                architectures = ["x86_64"]
```

Comment:
if the file isn't there, something is wrong. I suggest not catching this exception and letting the program error (no try/except).